app/src/main/index.js

```diff
@@ -232,6 +239,8 @@ async function startLCD(home, nodeIP) {
       reject()
       afterBooted(() => {
         if (mainWindow) {
+          // TODO unify/refactor logError and webContents.send
+          logError(`The ${SERVER_BINARY} rest-server (LCD) exited unplanned`)
```

Comment:
i know this is not a change from this PR - but why do we call this `SERVER_BINARY`?

Reply:
I guess it could be called LCD_BINARY. The SERVER part must refer to the REST server.

Reply:
i think a name change could reduce confusion. too many combinations of `server, cli, binary` to keep track of.